modules/administration-guide/pages/applying-the-github-oauth-app-secret.adoc

```diff
@@ -0,0 +1,56 @@
+:_content-type: PROCEDURE
+:description: Applying the GitHub OAuth App Secret
+:keywords: applying-the-github-oauth-app-secret, apply-the-github-oauth-app-secret, apply-github-oauth-app-secret, apply-secret, applying-secret, apply-a-secret, applying-a-secret
+:navtitle: Applying the GitHub OAuth App Secret
+// :page-aliases:
+
+[id="applying-the-github-oauth-app-secret_{context}"]
+= Applying the GitHub OAuth App Secret
+
+Enable the users of your {prod-short} instance to access a private Git repository that is hosted on GitHub:
+
+.Prerequisites
+* The repository owner has xref:user-guide:setting-up-the-github-oauth-app.adoc[set up the GitHub OAuth App].
+* The repository owner has given you the (1) *GitHub OAuth Client ID* and (2) *GitHub OAuth Client Secret*.
```

Comment:
Should be changed so that its not the repository owner, but the admin that performs this task

Reply:
Thank you.